### ISSUE TITLE
Choose for player at every startup

### ISSUE THREAD

**Issue by federicosintucci** [federicosintucci/Volumio_ios#13]
@edmw can you take a look at the default player selection? Now it asks for the selection at every startup ;)

ps: thanks for the huge work you're doing, the app is more responsive than ever!

**Comment by edmw** [comment]
Yes, it sometimes works for me (most of the time not). I think this is a race condition between the connectDefault call and the first getState. I'll have a look into it.

PS: Don't mention it! It's fun and I'd like to have a good app for Volumio myself :-)

**Comment by edmw** [comment]
Ok, really seems to be a race condition. I'll merge a fix later. With this fix I tried to unify the connect/disconnect handling for the various view controllers. Surely this will need some testing :-)
